classes/fields/pick.php

```diff
@@ -872,7 +872,7 @@ public function build_dfv_field_options( $options, $args ) {
 
 		$options[ $args->type . '_limit' ] = $limit;
 
-		$options[ 'ajax_data' ] = $this->build_dfv_autocomplate_ajax_data( $options, $args, $ajax );
+		$options[ 'ajax_data' ] = $this->build_dfv_autocomplete_ajax_data( $options, $args, $ajax );
```

Comment:
That's a weird diff

Reply:
I may have made some mention about reading glasses....